javascript/src/main/App.css

```diff
@@ -59,13 +59,13 @@ body {
   /* bottom = footer height */
 }
```

Comment:
Can we remove the margin here? It seems to from when the footer was absolutely positioned.

Reply:
I'm going to put this suggestion in the refactoring story too. (#276)